syft/serde/protobuf/proto.py

```diff
@@ -34,7 +35,7 @@
 from syft_proto.generic.pointers.v1.pointer_tensor_pb2 import PointerTensor as PointerTensorPB
 from syft_proto.messaging.v1.message_pb2 import ObjectMessage as ObjectMessagePB
 from syft_proto.messaging.v1.message_pb2 import TensorCommandMessage as CommandMessagePB
-from syft_proto.types.syft.v1.id_pb2 import Id as IdPB
```

Comment:
I suspect we may still need to import `Id` here too.

Reply:
It's not used anywhere... Why would we need to import it?

Reply:
Whoa, didn't realize this was the only place where `IdPB` was used so far. It will definitely be used more in the future, but we can put it back when we need it.

Reply:
It was imported but not even used...
But we definitely need to use that!